.github/CODEOWNERS

```
@@ -0,0 +1,27 @@
+#Note that despite github calling these "owners", a better term would be "alerts"
+#Everything in the cataclysm repository remains under a creative commons license, nobody owns the mods.
+#In fact, the code "owners" designation is meant to encourage third party
+#individuals to contribute to the mod, with the maintainers notified for reviews.
+
+/data/mods/Aftershock/ @Malecypse @John-Candlebury
+/data/mods/DinoMod/ @LyleSY
+/data/mods/Graphical_Overmap/ @Kilvoctu @Larwck
+/data/mods/MMA/ @Hymore246
+/data/mods/No_Fungi/ @I-am-Erk
+/data/mods/blazemod/ @SouP
+/data/mods/classic_zombies/ @I-am-Erk
+/data/mods/crt_expansion/ @Soupster89
+/data/mods/desertpack/ @davidpwbrown 
+/data/mods/generic_guns/ @tenmillimaster
+/data/mods/magiclysm/ @KorGgenT
+/data/mods/my_sweet_cataclysm/ @Fris0uman
+/data/mods/ruralbiome/ @I-am-Erk
+/data/mods/speedydex/ @KorGgenT
+/data/mods/stats_through_kills/ @KorGgenT
+
+magic_*.cpp @KorGgenT
```

```suggestion
magic*.cpp @KorGgenT
```
same for magic*.h
because there's also a magic.cpp and magic.h that will get excluded. sorry that i steered you a tiny bit wrong when i mentioned it.